waku/node/v2/waku_types.nim

```diff
@@ -5,6 +5,7 @@
 import
   std/tables,
   chronos,
+  random,
```

Comment:
We should not be using stdlib random for this. Its random implementation is considered insecure. 
Can use libp2p crypto or eth keys rng for this (which is actually using the same one, the one of bearssl).

e.g. https://github.com/status-im/nim-waku/blob/a8dbf8a7b6fbf6a8dcdea0d8308236dcdcb6515e/waku/node/v2/start_network2.nim#L32

Reply:
https://github.com/status-im/nim-waku/issues/187